examples/image/create/main.go

```diff
@@ -16,10 +16,10 @@ func main() {
 
 	client := image.NewClient(s)
 	resp, err := client.Create(ctx, &image.CreateParams{
-		N:      3,
-		Prompt: "a cute baby",
-		Size:   "1024x1024",
-		Format: "b64_json",
+		N:              3,
+		Prompt:         "a cute baby",
+		Size:           "1024x1024",
+		ResponseFormat: "b64_json",
```

Comment:
Do we need to break this?

Reply:
Could you please explain what do you mean by "break it"? do you mean into separate task or PR? or into set of values like enum?

Reply:
Oh yes I got it now! I revert the field ResponseFormat to Format. I also reverted Created to CreatedAt.